README.md

```diff
@@ -1 +1,161 @@
-# Hack-Summit
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🛠 Walkthough](#walkthough)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 HakcSummit 2023 <a name="about-project"></a>
+
+**HackSummit 2023** is a capstone project for HTML CSS and JavaScript module at microverse
+
+## 🛠 Built With <a name="built-with"> HTML CSS & Java Script</a>
+
+- Video to walkthrough of this project can be found [here]("https://www.loom.com/share/354d06ce5b2d4e83a8d2eef21ad1cd48")
+
+## 🛠 Walkthough <a name="walkthough"> https://www.loom.com/share/354d06ce5b2d4e83a8d2eef21ad1cd48</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML CSS & Java Script
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Presents the Home page for HackSummit 2023**
+- **Presents the About page for HackSummit 2023**
+- **Displays Speakers List with more and less button functionality**
+- **Displays Mobile Menu**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> The live demo link to this project can be found at: 
+
+- [Live Demo Link](https://kalbek.github.io/Capstone-1-Hack-Summit/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a web browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project using the cloned repo or download and use the files to your local machine.
+
+### Usage
+
+To run the project open the index.html file in your web browser
+
+### Run tests
+
+No test are available
+
+### Deployment
+
+You can deploy this project through Github Pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] as you know you did a mistake in the project submission kindly consider adding a descriptive way  for this Getting Started Section  cause currently  its not giving clear guidelines on how to access your project locally 😨, The "Getting Started" section in a readme file is typically used to provide instructions and guidance to users who want to start using a project or application. It should include a clear set of steps to help users get the project up and running quickly and efficiently.
eg: add the right repo link to clone when checking your project and also how to get it on live server as your project is being accessed through pages



Kindly provide a clear guide for a user to get started with your project 🙏